Makefile

```diff
@@ -678,13 +680,20 @@ transifex-init: .build/dev-requirements.timestamp .tx/config \
 	.build/python-venv/bin/tx pull -l $* --force
 	$(TOUCHBACK_TXRC)
 
+.PRECIOUS: .build/locale/%/LC_MESSAGES/demo.po
+.build/locale/%/LC_MESSAGES/demo.po:
+	mkdir -p $(dir $@)
+	wget -O $@ https://raw.githubusercontent.com/camptocamp/demo_geomapfish/master/demo/locale/$*/LC_MESSAGES/demo-client.po
```

Comment:
I hope that doesn't create some problems like in https://github.com/camptocamp/ngeo/blob/master/buildtools/cleanup-ghpages.py#L14